src/sentry/utils/json.py

```diff
@@ -40,6 +41,8 @@ def better_default_encoder(o):
         return six.text_type(o)
     elif isinstance(o, Enum):
         return o.value
+    elif isinstance(o, BitHandler):
```

Comment:
this is overkill and i could back it out if we want

Reply:
Nah, I think this is fine.